conda-launchers/recipe.yaml

```diff
@@ -0,0 +1,153 @@
+context:
+  name: conda-launchers
+  version: "25.0.0"
```

Comment:
Not sure what version number is properly used here.

Reply:
For now let's use the same as in `meta.yaml`